Dockerfile

```diff
@@ -76,4 +76,7 @@ ENV NODE_ENV=production
 COPY --from=npm-prod /app/ .
 COPY --from=build-prod /app/build/ build/
 
+USER node
```

Comment:
Guess we do want this after things have been copied in?

Reply:
And no need to replicate on `dev`.

Reply:
Just read the guidance, makes sense to only include it at the end of `prod`.

Reply:
`COPY` doesn't follow `USER` for file ownership, but `RUN npm ...` will; so my default is to switch at the end as I'm not very concerned about `root` in the build phase. Can be moved up to just after `npm run build` but not very useful to do so.